src/Loader.php

```diff
@@ -192,6 +192,22 @@ public static function is_onboarding_enabled() {
 		return false;
 	}
 
+	/**
+	 * Determines if a minified JS file should be served.
+	 *
+	 * @param  boolean $script_debug Only serve unminified files if script debug is on.
+	 * @return boolean If js asset should use minified version.
+	 */
+	public static function should_use_minified_js_file( $script_debug ) {
+		// un-minified files are only shipped in non-core versions of wc-admin, return true if unminified files are not available.
+		if ( ! self::is_feature_enabled( 'unminified-js' ) ) {
```

Comment:
Theoretically someone could still make this break by filtering `woocommerce_admin_features` to include `unminified-js` on a site running `core` - but I'd say that is super edge case / crazy pills.